package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "dp-design-system",
+  "version": "1.0.0",
+  "description": "Digital Publishing (ONS) Design System",
+  "main": "",
+  "scripts": {
+    "build-css": "mkdir -pv dist/css && node-sass --output-style compressed scss/main.scss dist/css/main.css",
+    "build": "npm run build-css",
+    "watch-css": "onchange 'scss' -e 'scss/work-in-progress' -e 'scss/work-in-progress.scss' -- npm run build-css",
+    "server": "http-server -p ${PORT:-9001} --cors",
+    "dev": "npm run watch-css & npm run server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ONSdigital/dp-design-system.git"
+  },
+  "author": "Digital Publishing (ONS)",
+  "license": "Open Government License v3.0",
+  "bugs": {
+    "url": "https://github.com/ONSdigital/dp-design-system/issues"
+  },
+  "homepage": "https://github.com/ONSdigital/dp-design-system#readme",
+  "dependencies": {
+    "http-server": "^0.12.3",
+    "node-sass": "^5.0.0",
```

Comment:
(If we do decide on Sass): We ought to look at using another library here as `node-sass`, as of a few months ago, is now [deprecated](https://sass-lang.com/blog/libsass-is-deprecated) (along with its underlying package, `libsass`). Recommendations from the team there are to use [`dart-sass`](https://github.com/sass/dart-sass) instead as it's the canonical implementation. Changing `node-sass` to `sass` is all that's needed, but we can discuss this in the fundamental principles meeting next week

Reply:
Done, some of our npm scripts needed changing too. On this subject of node, I have now gone for the latest long time support version. Although this can be changed.

Reply:
Cool thanks, sounds good to stick with LTS to me